modules/local/create_bed.nf

```diff
@@ -0,0 +1,31 @@
+process CREATE_BED {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "conda-forge::python=3.9.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.9--1' :
+        'quay.io/biocontainers/python:3.9--1' }"
+
+    input:
+    tuple val(meta), path (fasta)      //path to fasta windows file
```

Comment:
The output of fasta_windows (and input to this create_bed module) is a TSV file, not a Fasta file